tensorboard/webapp/plugins/npmi/util/metric_type.ts

```diff
@@ -24,7 +26,26 @@ export function metricIsNpmi(metric: string): boolean {
   return metric.startsWith('nPMI');
 }
 
+export function metricIsNpmiAndNotDiff(metric: string): boolean {
+  return metric.startsWith('nPMI@');
+}
+
 export function stripMetricString(metricString: string): string {
   const strippedString = metricString.split('@', 2)[1];
   return strippedString;
 }
+
+export function addSortingSymbol(
+  metricString: string,
+  sorting: {metric: string; order: SortingOrder}
+): string {
+  let result = metricString;
+  if (metricString === sorting.metric) {
+    if (sorting.order === SortingOrder.DOWN) {
+      result = result + ' 🠧';
```

Comment:
Hmm, I am not sure if it is GitHub or my computer that is screwed up but this unicode does not render properly on my macOS. 
![image](https://user-images.githubusercontent.com/2547313/91523775-7439f700-e8b2-11ea-9170-3a40b883e8c7.png)